src/app.controller.auth.ts

```
@@ -140,3 +185,52 @@
 	}
 }
+
+function getAuthCookieOpts(isRefreshToken: boolean): CookieOptions {
+	if (Env.IsProd() && !process.env.AUTH_COOKIES_DOMAIN && !process.env.BASE_URL_API) {
+		throw new Error('AUTH_COOKIES_DOMAIN or BASE_URL_API must be set in production')
+	}
+
+	let domain
+	try {
+		domain = process.env.AUTH_COOKIES_DOMAIN || new URL(process.env.BASE_URL_API).hostname
+	} catch (e) {
```

ESLint

[error] 'e' is defined but never used.

(@typescript-eslint/no-unused-vars)